.github/workflows/templates_python.yml

```diff
@@ -37,6 +37,9 @@ jobs:
           - {name: "pyansys-advanced-flit", template: "pyansys_advanced", build-system: "flit", outdir: "pyansys_a0/pyproduct-library"}
           - {name: "pyansys-advanced-poetry", template: "pyansys_advanced", build-system: "poetry", outdir: "pyansys_a1/pyproduct-library"}
           - {name: "pyansys-advanced-setuptools", template: "pyansys_advanced", build-system: "setuptools", outdir: "pyansys_a2/pyproduct-library"}
+          
+          # PyAnsys OpenAPI Client template tests
+          - {name: "pyansys-openapi-client", template: "pyansys_openapi_client", build-system: "flit", outdir: "pyansys_o0/pyproduct-library-openapi"}
```

Comment:
I guess we can exclude this template from the CI so it can get merged, right @da1910?

Reply:
For the moment that's probably the best way forward. I would like to get an end-to-end test of the library, and a rendered example branch, but I haven't had much time recently.

Reply:
I've done so, I think this should be good to go for the moment